### ISSUE TITLE
Client: listening for incoming traffic

### ISSUE THREAD

**Issue by MarioCode** [swiftsocket/SwiftSocket#76]
The application uses a few view, where is used one common socket (define in MainTabBarView). At a different time from the server should come response/request. The question is where and how they handle requests, If I do not expect them?
Now I send it and immediately the request comes:

```
switch client.send (string: "AUTH" + login + " /s " + pass.md5 ()) {
             case .success:
                 return client.readResponse ()
             case .failure (let error):
                 return nil
             }
```

But how to catch the received response, will the client work as a server and process them at different points in time?

#### In other words, from the server come user messages, notifications, and where and how to catch them without a switch-cases, I can not understand. Or this library does not provide for this?

**Comment by danshevluk** [comment]
Here is a similar issue: https://github.com/swiftsocket/SwiftSocket/issues/70

You can create a dedicated dispatch queue for reading with infinite reading loop inside. 
This is the only way to implement this behaviour in `SwiftSocket`

**Comment by MarioCode** [comment]
Yes, I didn't notice. Thanks.
